hack/test/e2e-firecracker.sh

```diff
@@ -16,9 +28,9 @@ function create_cluster {
     --memory 2048 \
     --cpus 2.0 \
     --cidr 172.20.0.0/24 \
-    --init-node-as-endpoint \
```

Comment:
this isn't required anymore, as firecracker LB running on the bridge provides default endpoint